src/raft/recv_install_snapshot.c

```diff
@@ -541,19 +541,19 @@ static void work_fill_follower(struct follower *follower)
 		break;
 	}
 	work_init(&follower->work);
-	// sm_to_sm_obs(&follower->sm, &follower->work.sm);
+	sm_to_sm_obs(&follower->sm, &follower->work.sm);
 }
 
 static void rpc_fill_leader(struct leader *leader)
 {
 	rpc_init(&leader->rpc);
-	// sm_to_sm_obs(&leader->sm, &leader->rpc.sm);
+	sm_to_sm_obs(&leader->sm, &leader->rpc.sm);
 }
 
 static void rpc_fill_follower(struct follower *follower)
 {
 	rpc_init(&follower->rpc);
-	// sm_to_sm_obs(&follower->sm, &follower->rpc.sm);
+	sm_to_sm_obs(&follower->sm, &follower->rpc.sm);
```

Comment:
For future: let's think about better function name for `sm_to_sm_obs()`.

Reply:
Changed to `sm_relate`, I am open to better suggestions :)